### PR TITLE
par.vals check

### PR DESCRIPTION
**Pull request by Dotterbart**
par.vals and appropriate LearnerParam compared and checked

### CODE REVIEW

**Comment by larskotthoff** [comment]
Thanks for the PR. Could you explain please what the purpose is? As far as I can see, it's only adding FIXME comments.

**Comment by berndbischl** [comment]
Please always add readable explanations in your PR, what the purpose is. Especially when there is no referenced issue in the background. Otherwise he have no idea what this is can cannot check and review.

**Comment by Dotterbart** [comment]
Ok, sorry. I checked all the learners for cases, where the par.vals argument(s) and the defaults of the appropriate make*LearnerParams are the same. In these cases normally either the make*LearnerParam default is not the same default as in the used function or the par.vals argument is redundant.

**Comment by larskotthoff** [comment]
Thanks for the explanation. Could you fix these issues then please instead of just adding a comment?

**Comment by jakob-r** [comment]
I kindly asked @Dotterbart to look for contradictions in the par.vals and told him to comment on a fork so we can easily check the spotted issues.
Because they are also part of what I work on in branch [fix_344_default_par_vals](https://github.com/mlr-org/mlr/compare/fix_344_default_par_vals?expand=1) (Issue #344) I didn't insist on corrections because they will only lead to more merging conflicts.
In other words: This PR is not intended to be merged

**Comment by berndbischl** [comment]
OK. 
1) Thx.
2) But please dont be to "lazy" then to communicate this and say it in the PR. Otherwise it leads to confusion on our side.

**Comment by berndbischl** [comment]
@jakob-r 
this was your idea, please tell us what to do with this PR. shall I close this? because it is 6 months old and not moving forward.

**Comment by jakob-r** [comment]
1. Do we want to make an Issue out of every FIXME? _IMO: No_
2. Do we want to lose that Information? _IMO: No_
3. Do we want to fix everything no? _Yes, but I don't have the time or the intrinsic urge to do it_

Ergo: Merge and we are fine :) 
Point 3. will come up at specific times and it will get fixed with the time.

**Comment by berndbischl** [comment]
ok, then lets please merge it in. but @Dotterbart has to rebase this on master. the branch has conflicts

**Comment by jakob-r** [comment]
I did it manually as @Dotterbart unfortunately already deleted his fork.
